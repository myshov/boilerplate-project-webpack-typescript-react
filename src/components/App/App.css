body, html {
    padding: 0;
    margin: 0;
}

.app {
  font-family: Arial, Helvetica, sans-serif;

  display: flex;
  justify-content: center;
}
