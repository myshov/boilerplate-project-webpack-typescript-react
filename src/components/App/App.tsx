import React from 'react';
import { hot } from 'react-hot-loader/root';

import styles from './App.css';

function App(): JSX.Element {
  return (
    <div className={styles.app}>
      <h1>App</h1>
    </div>
  );
}

export default hot(App);
