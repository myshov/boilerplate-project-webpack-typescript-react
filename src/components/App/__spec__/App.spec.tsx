import React from 'react';
import { shallow } from 'enzyme';

import App from '../App';

describe('Component "App"', () => {
  it('should render correctly', () => {
    const component = shallow(<App />);
    // console.log(component.debug())
    expect(component).toMatchSnapshot();
  });
});
