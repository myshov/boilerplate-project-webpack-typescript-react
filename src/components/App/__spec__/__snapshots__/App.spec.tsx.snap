// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component "App" should render correctly 1`] = `
<div
  className="App"
>
  <h1>
    App
  </h1>
</div>
`;
